NEWS.md

```diff
@@ -0,0 +1,10 @@
+# admiraltest 1.1.0
+
+- Extending the test data offering by adding PK datasets - PC and PP (#1) 
+- Adding Oncology test data - RS, TU, TR, SUPPTR (#2)
+- Adding SV from CDISC pilot project (#20)
+
+
+# admiraltest 1.0.0
```

Comment:
The version released at CRAN is 0.1.2.

Thus we should change this heading and maybe use a different version number for the new version, e.g., 0.2.0?

Reply:
yes you're right, updated now and added entries to explain 0.1.1 and 0.1.2